migrator/migration.py

```diff
@@ -109,6 +116,16 @@ def iam_certificate_id(self):
             "IAMCertificateId"
         ]
 
+    @property
+    def iam_certificate_name(self):
+        return self.cloudfront_distribution_config["ViewerCertificate"]["Certificate"]
+
+    @property
+    def iam_certificate_arn(self):
+        return self.cloudfront_distribution_config["ViewerCertificate"][
+            "ACMCertificateArn"
```

Comment:
we don't use ACM.
To get the cert ARN I think we have to get the cert information from IAM using the certificate ID

IAM server certificates are great.

Reply:
🤪

Reply:
It looks like we could use the `IAMCertificateId` from the `ViewerCertificate` to then run [`get_server_certificate(**kwargs)`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.get_server_certificate) in IAM?

Reply:
looks like that wants the certificate name. 
As far as I can tell, you're going to need to use list_server_certificates and loop through them.

Reply:
Yeah I saw that with one of the other methods and was hoping that ID could be passed in with this method.  Either, this is a place to start, thank you! :-)